src/preamble.js

```diff
@@ -740,20 +740,19 @@ if (!isDataURI(wasmBinaryFile)) {
   wasmBinaryFile = locateFile(wasmBinaryFile);
 }
 
-function getBinary() {
+function getBinary(file) {
   try {
     if (wasmBinary) {
-      return new Uint8Array(wasmBinary);
+      abort("BADBADBAD");
```

Comment:
TODO: Delete this if it doesn't mess up the tests. Otherwise, figure out what `wasmBinary` is supposed to be and how to refactor it now that `getBinary` takes a file parameter.